docs/source/documentation/diagrams/architecture-diagrams.html.md.erb

```diff
@@ -0,0 +1,33 @@
+# Data Platform - Architectural Diagrams
+
+## C4 Model Diagrams
+
+In the [C4 model](https://c4model.com/), the four C's represent [context](https://c4model.com/#SystemContextDiagram), [containers](https://c4model.com/#ContainerDiagram), [components](https://c4model.com/#ComponentDiagram), and [code](https://c4model.com/#CodeDiagram). These different levels help you to tell the right story to the right audience.
```

Comment:
I'd clarify the story and audience here.

'These different levels help you tell the right story to the right audience. In this case, for the Data Platform to its prospective users respectively.' [or to whoever the audience is if not them]